### PR TITLE
UI Improvements

### PR DESCRIPTION
**Pull request by asaadmahmoodspin**
- Improving image modals.
- Improving modal functionality to apply classes on the correct modal.
- Improving online indicator to only go the same as text color if the online indicator and active bg are the same.

### CODE REVIEW

File: web/react/components/channel_loader.jsx [hunk 1]
```diff
@@ -104,12 +104,6 @@ export default class ChannelLoader extends React.Component {
             }
         });
 
-        /* Setup modal events */
-        $('.modal').on('show.bs.modal', function onShow() {
-            $('.modal-body').css('overflow-y', 'auto');
-            $('.modal-body').css('max-height', $(window).height() * 0.7);
-        });
```
Comment (jwilander) [inline web/react/components/channel_loader.jsx]: Do giant images have their size controlled somehow?

**Comment by crspeller** [comment]
+1 Other than Joram's concern.

**Comment by asaadmahmoodspin** [comment]
Yes, inside:
https://github.com/asaadmahmoodspin/platform/blob/ui-improvements/web/react/components/view_image.jsx

File: web/react/components/view_image.jsx [hunk 2]
```diff
@@ -195,6 +195,7 @@ export default class ViewImageModal extends React.Component {
                         target='_blank'
                     >
                         <img
+                            style={{maxHeight: this.state.imgHeight}}
```
Comment (asaadmahmoodspin) [inline web/react/components/view_image.jsx]: This to control image height.

File: web/react/components/view_image.jsx [hunk 3]
```diff
@@ -210,6 +211,7 @@ export default class ViewImageModal extends React.Component {
 
                 content = (
                     <video
+                        style={{maxHeight: this.state.imgHeight}}
```
Comment (asaadmahmoodspin) [inline web/react/components/view_image.jsx]: This to control video height.

**Comment by asaadmahmoodspin** [comment]
Hey guys have a look at this PR.

**Pull request closed without merging**